tests/test_models.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+from cmsplugin_articles_ai.factories import TagFactory
+
+
+def test_tag_factory_slug():
```

Comment:
I refactored slug into field instead of a property. Therefore the slug isn't set in this scenario. Made this test just to test that the lazy assignment in the factory works.